src/ui/units/dash/modules/helpers.ts

```diff
@@ -176,7 +176,7 @@ export function addOperationForValue(args: {
 
 export function getChartEditLink(entryId: string) {
     return Utils.isEnabledFeature(Feature.UseNavigation)
-        ? `${window.DL.endpoints.charts}/navigation/${entryId}`
+        ? `${window.DL.endpoints.charts}/navigate/${entryId}`
```

Comment:
I think here should be `/navigate/${entryId}` or `${window.location.origin}/navigate/${entryId}`

Reply:
Yep, I fix it a second ago 🙏

/navigation - for old behaviour with enabled UseNavigation flag
/navigate - for new route wuth support not only for wizard

Reply:
Are you sure that we need window.DL.endpoints.charts for new route?

Reply:
No, you are right